packages/feedback-service/src/feedbacks/resolver.ts

```diff
@@ -94,15 +94,13 @@ const FeedbackResolver = {
 
       const userQuery = FeedbackHelper.buildUserQuery([userFeedback.createdBy]);
       userData = await FeedbackHelper.getUserProfiles(userQuery);
-      appList = await FeedbackHelper.listApps();
 
-      const app = appList.sort((a, b) => b.path.localeCompare(a.path))
-        .find(({ path }) => userFeedback.stackInfo.path.includes(path));
-      if (!app) {
-        throw new Error('App not found. Please visit developer-console');
+      const { projectId } = userFeedback;
+      if (!projectId) {
+        throw new Error('ProjectId not found. Please visit developer-console');
       }
       const feedbackConfig = await FeedbackConfig.find({
```

Comment:
Use `findOne` when looking up a singular record.
```suggestion
      const feedbackConfig = await FeedbackConfig.findOne({
```

Reply:
We would have a single feedbackConfig as the projectId is already unique

Reply:
All the more reason to use `findOne`

Reply:
![Screenshot 2023-01-30 at 2 12 49 PM](https://user-images.githubusercontent.com/12994292/215428700-e10cacdf-bd76-444a-a0c5-092c2fb7a987.png)
Ohh you mean for the code I did not touch. Okay got it.

Reply:
Updated